backend/code_review_backend/issues/migrations/0015_remove_repository_phid_alter_repository_id.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.1.2 on 2024-11-18 15:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("issues", "0014_unique_hash"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="repository",
+            name="phid",
+        ),
+        migrations.AlterField(
```

Comment:
Does the current value of the counter need to be set? From the Treeherder MySQL to PostgreSQL migration I remember the primary keys (sometimes?) having low values (starting at the initial value?) and storing new rows failed because of this. Is Django smart enough to calculate the current value?

Reply:
Good catch, you're right, I tested on our local copy of prod data, and got an error because the sequence has not been initialized:

```python
>>> from code_review_backend.issues.models import *
>>> from django.db.models import *
>>> Repository.objects.all().aggregate(max=Max('id'))
{'max': 1001}
>>> Repository.objects.create(url='http://test', slug="test")
Traceback (most recent call last):
  File "/home/bastien/Envs/cr-backend/lib/python3.10/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "issues_repository_pkey"
DETAIL:  Key (id)=(1) already exists.
```

@vrigal you can add for postgresql the following sql query in the migration
```
SELECT setval(pg_get_serial_sequence('issues_repository', 'id'), coalesce(max(id)+1, 0), false) FROM issues_repository;
 setval 
```